From 45b840275a5f5d3a3e4803c72c02a1db7fe1927d Mon Sep 17 00:00:00 2001
From: Laurent Vivier <lvivier@redhat.com>
Date: Mon, 6 Jul 2020 09:41:16 +0000
Subject: [PATCH] redhat: define hw_compat_8_2

RH-Author: Laurent Vivier <lvivier@redhat.com>
Message-id: <20200706104117.219174-2-lvivier@redhat.com>
Patchwork-id: 97903
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH v2 1/2] redhat: define hw_compat_8_2
Bugzilla: 1853265
RH-Acked-by: David Gibson <dgibson@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Greg Kurz <gkurz@redhat.com>

Signed-off-by: Laurent Vivier <lvivier@redhat.com>
Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
   For minor fix
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/core/machine.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index 5476af98e1..6d17d9938d 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -54,6 +54,17 @@ GlobalProperty hw_compat_rhel_8_2[] = {
     { "qxl-vga", "revision", "4" },
     /* hw_compat_rhel_8_2 from hw_compat_4_2 */
     { "fw_cfg", "acpi-mr-restore", "false" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "virtio-balloon-device", "page-poison", "false" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "vmport", "x-read-set-eax", "off" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "vmport", "x-signal-unsupported-cmd", "off" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "vmport", "x-report-vmx-type", "off" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "vmport", "x-cmds-v2", "off" },
+
 };
 const size_t hw_compat_rhel_8_2_len = G_N_ELEMENTS(hw_compat_rhel_8_2);
 /*
-- 
2.27.0

